helloextend-protection/includes/class-helloextend-protection-orders.php

```
@@ -175,6 +175,12 @@ public function create_update_order(string $order_id, array $order = null)
             $order = wc_get_order($order_id);
         }
         $order_data = $order->get_data();
+        $items = $order->get_items();
+        $product_id_array = array();
```

PHPMD (2.15.0)

The variable $product_id_array is not named in camelCase. (Controversial Rules)

(CamelCaseVariableName)